src/platforms/common/data-management/sensitive-data/attachment-scrubbing.mdx

```diff
@@ -0,0 +1,92 @@
+---
+title: "Attachment Scrubbing"
+sidebar_order: 4
+keywords: ["pii", "gdpr", "personally identifiable data", "compliance", "minidump"]
+---
+
+
+Events can have attachments which may also contain PII.
+[Minidumps](/platforms/native/guides/minidumps) are a common attachment type for native crash reports which can contain PII in various sections of the file.
+
+## Data scrubbing methods in attachments
+
+When scrubbing attachments, and especially minidumps, it is often important to be able to do this without destroying the original file format.
+This is done by understading enough of the file format and using custom selectors which will try and only let you modify certain parts of the file, [see below](#source-selectors) for more information on the source selectors.
+
+However when actually modifying some data in the file, we can not modify the length of the file, this has implications for how the [Methods](../advanced-datascrubbing/#methods) behave:
+
+- Any modification which results in a replacement text which is **longer** than the original will be **truncated**.
+- Any modification which results in a replacement text which is **shorter** than the original will be **padded** using `x` as padding character.
+
+Here is how this works specifically for each method:
+
+- *Remove*: As the entire field is removed, it is essentially replaced with the padding character.
+- *Mask*: This behaves as normal, all data is replaced with `*` masking character.
+- *Hash*: Hasing happens as usual, however the **binary representation** of the match is hashed, regardless of the encoding the match was made in.  The resulting hash is padded or truncated as described above.
+- *Replace*: This behaves like normal, with the replacement being padded or truncated as described above.
+
+## Source selectors
+
+All attachments can be select using the `$attachments` [Value Type](../advanced-datascrubbing/#value-types) selector at the root of the selector, this can be followed by the specific filename itself.
```

Comment:
```suggestion
All attachments can be selected using the `$attachments` [Value Type](../advanced-datascrubbing/#value-types) selector at the root of the selector, this can be followed by the specific filename itself.
```